ccmlib/node.py

```diff
@@ -1121,7 +1121,8 @@ def stress(self, stress_options=None, capture_output=False, **kwargs):
                 p = subprocess.Popen(args, cwd=common.parse_path(stress),
                                      **kwargs)
                 stdout, stderr = None, None
-            p.wait()
+            return_code = p.wait()
+            assert return_code == 0, "Stress command exited with error code: %s" % return_code
```

Comment:
Can we receive more informative error message, then number?

Reply:
Yep, good catch

Reply:
fixed